modules/core/src/persistence.cpp

```diff
@@ -2398,7 +2398,12 @@ std::string FileNode::string() const
 {
     const uchar* p = ptr();
     if( !p || (*p & TYPE_MASK) != STRING )
-        return std::string();
+    {
+        if ( (*p & TYPE_MASK) == NONE )
+            return std::string("None");
```

Comment:
At first I think this might be confusing in Python and an empty string is a better chose. However in Python `str(None)` is also a string `"None"`.

Reply:
btw, printing a json object of null also outputs `None` (Python keyword). m.b. we need to do something to python bindings.